craft_parts/parts.py

```diff
@@ -433,9 +443,11 @@ class PartSpec(BaseModel):
         default=None,
         description="The commands to run instead of the default behavior of the stage step.",
         examples=[
-            '''|
-              craftctl default
-              chown -R 499 "${CRAFT_PART_INSTALL}/entrypoint.sh"'''
+            textwrap.dedent(
+                '''\
+                |
+                craftctl default
+                chown -R 499 "${CRAFT_PART_INSTALL}/entrypoint.sh'''
         ],
```

Comment:
```suggestion
            textwrap.dedent(
                '''\
                |
                craftctl default
                chown -R 499 "${CRAFT_PART_INSTALL}/entrypoint.sh'''
            )    
        ],
```
Missing closing parenthesis